emgapi/models.py

```diff
@@ -640,6 +641,97 @@ class Meta:
         unique_together = (('study', 'sample'),)
 
 
+class SuperStudyQuerySet(BaseQuerySet):
+    pass
+
+
+class SuperStudyManager(models.Manager):
+
+    def get_queryset(self):
+        return SuperStudyQuerySet(self.model, using=self._db) \
+            .annotate(biomes_count=Count('biomes', distinct=True))
+
+    def available(self, request, prefetch=False):
+        queryset = self.get_queryset()
+        if prefetch:
+            queryset = queryset.prefetch_related(
+                Prefetch('flagship_studies', queryset=Study.objects.available(request)),
+                Prefetch('biome', queryset=Biome.objects.all()),
+            )
+        return queryset
+
+
+class SuperStudy(models.Model):
+    """
+    Aggregation of studies.
+    Each Super Study will have multiples Studies under 2 categories:
+    - Flaship Projects, those that are directly related to the Super Study
+    - Related Projects, the studies that share the biome with the Super Study
+    """
+    super_study_id = models.AutoField(db_column='STUDY_ID',
+                                      primary_key=True)
+    title = models.CharField(db_column='TITLE',
+                             max_length=100, blank=True, null=True)
```

Comment:
```suggestion
                             max_length=100)
```
This should be required, otherwise the collection doesn't really have any basic designation

Reply:
Yep, good catch :)